sysbox-in-docker/Dockerfile.ubuntu-eoan

```diff
@@ -0,0 +1,78 @@
+#
```

Comment:
We should talk about how to handle EOL'd releases. In my case i've been leaving existing Dockerfiles untouched, but i'm not creating new ones to cover EOL'd releases as we increase functionality. The main reason is that it's hard to test these unsupported releases as there's no even a possibility to build their associated Docker images.

Reply:
My dev machine is eoan, so I needed that version. But you are right, we need to discuss how to EOL releases.